datafusion/sqllogictest/test_files/expr.slt

```diff
@@ -939,6 +939,153 @@ SELECT date_part('nanosecond', '2020-09-08T12:00:12.12345678+00:00')
 ----
 12123456780
 
+# test_date_part_time
```

Comment:
Is it also worth testing the `EXTRACT (...)` syntax too ?

like `EXTRACT( hour from arrow_cast('23:32:50'::time, 'Time32(Second)')))`?

Reply:
Good point, added :+1: